apps/frontend-v3/app/(app)/testooors/page.tsx

```diff
@@ -0,0 +1,41 @@
+'use client'
+import { HStack, Heading, Link, Stack, Text, VStack } from '@chakra-ui/react'
+import FadeInOnView from '@repo/lib/shared/components/containers/FadeInOnView'
+import NextLink from 'next/link'
+
+export default function TestooorV3Pools() {
```

Comment:
Replacing old debug page with specific page for zen testooors (that will be deleted after the first tests)